src/screens/Signup/StepCaptcha.tsx

```diff
@@ -62,33 +44,32 @@ export function Step3({
   )
 
   const onError = React.useCallback(() => {
-    uiDispatch({
-      type: 'set-error',
+    dispatch({
+      type: 'setError',
       value: _(msg`Error receiving captcha response.`),
     })
-  }, [_, uiDispatch])
+  }, [_, dispatch])
 
   return (
-    <View>
-      <StepHeader uiState={uiState} title={_(msg`Complete the challenge`)} />
+    <Animated.View
+      style={[a.gap_lg]}
+      entering={FadeInRight}
+      exiting={FadeOutLeft}>
```

Comment:
In the login flow, I disabled this on web. Do we want to do the same here? (or we could reenable for the login)

Reply:
I am leaning towards disable on web

Reply:
Just used `ScreenTransition`. Just had to change where I used it. Reanimated layout transitions confuse me sometimes :/